.github/workflows/test-install.yml

```diff
@@ -158,13 +158,21 @@ jobs:
       with:
         conda-channels: conda-forge
 
+    - run: conda update conda
     - run: conda --version
 
+    # Use mamba because conda is running out of memory
+    - run: conda install mamba -n base -c conda-forge
+    - run: mamba --version
+
+    # Temporary workaround: https://github.com/mamba-org/mamba/issues/488
+    - run: rm /usr/share/miniconda/pkgs/cache/*.json
+
```

Comment:
I hadn't looked into this no. Do we want to move in this direction though? The current change to me is a temporary workaround. The point of the test is to verify that one of the suggested installation methods in the docs, `conda`, works as expected. Ideally we would fix the workaround and having to use mamba in the near future, right? If you think it is not that important and prefer to use the action, that is fine by me as well.

Reply:
But it is not guaranteed to work the same as conda, is it? So it might be possible that the install works with mamba but fails with conda. So we should then suggest in the docs to use mamba instead of conda.